Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
There is no problem in having your Dockerfile in another folder. In the Docker guidelines for FIWARE the recommendation is to use the 'docker' folder 

thanks